packages/lesmis/src/components/EntityVitalsItem.js

```diff
@@ -102,7 +102,7 @@ export const EntityVitalsItem = ({ name, value, icon, isLoading }) => (
 
 EntityVitalsItem.propTypes = {
   name: PropTypes.string.isRequired,
-  value: PropTypes.string,
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
This just fixes a console error in lesmis, just sneaking it in alongside other work